src/main/scala/edu/ie3/powerFactory2psdm/converter/FixedFeedInConverter.scala

```diff
@@ -0,0 +1,52 @@
+/*
+ * © 2021. TU Dortmund University,
+ * Institute of Energy Systems, Energy Efficiency and Energy Economics,
+ * Research group Distribution grid planning and operation
+ */
+
+package edu.ie3.powerFactory2psdm.converter
+
+import edu.ie3.datamodel.models.input.NodeInput
+import edu.ie3.datamodel.models.input.system.FixedFeedInInput
+import edu.ie3.datamodel.models.input.system.characteristic.CosPhiFixed
+import edu.ie3.powerFactory2psdm.exception.pf.ConversionException
+import edu.ie3.powerFactory2psdm.model.powerfactory.StaticGenerator
+import edu.ie3.util.quantities.PowerSystemUnits.MEGAVOLTAMPERE
+import tech.units.indriya.quantity.Quantities
+
+import java.util.{Locale, UUID}
+
+object FixedFeedInConverter {
+
+  /**
+    * Converts a static generator to a [[FixedFeedInInput]]
+    *
+    * @param input generator to convert
+    * @param node node the static generator is connected to
+    * @return a fixed feed-in
+    */
+  def convert(input: StaticGenerator, node: NodeInput): FixedFeedInInput = {
+
+    val cosPhi = input.indCapFlag match {
+      case 0 => input.cosPhi
+      case 1 => -input.cosPhi
+      case _ =>
+        throw ConversionException(
+          "The leading/lagging specifier should be either 0 or 1 - I am confused!"
+        )
+    }
+    val varCharacteristicString =
+      "cosPhiFixed:{(0.0,%#.2f)}".formatLocal(Locale.ENGLISH, input.cosPhi)
+    val s = Quantities.getQuantity(input.sRated, MEGAVOLTAMPERE)
```

Comment:
Don't you need to account here for the PQS-Prefix specifications, I have seen in the other PR?

Reply:
The pqs prefixes only apply to loads and asynchronous machines.. please don't ask me why 😄

Reply:
Why? :joy:

Reply:
![image](https://user-images.githubusercontent.com/63147366/130618935-51ffdbad-e830-4b4c-b248-bc3d35a359da.png)

because they like messing with people 😄